src/components/link/link.ts

```diff
@@ -76,6 +97,12 @@ class BXLink extends FocusMixin(LitElement) {
   @property({ reflect: true })
   rel!: string;
 
+  /**
+   * Link size.
+   */
+  @property({ reflect: true })
+  size!: LINK_SIZE.REGULAR;
```

Comment:
This slipped through the crack of my review, my apologies. We use the regular version as the initial value, instead of the type:

```suggestion
  size = LINK_SIZE.REGULAR;
```

Reply:
Got it, thanks @asudoh!